pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -271,7 +281,7 @@ public int create(MercuryOrderDto mercuryOrderDto, String messageJson) {
 
         });
         if (results.resultException != null) {
-            throw new RuntimeException("Error inserting mercury order ", results.resultException);
+            throw new DsmInternalError("Error inserting mercury order ", results.resultException);
```

Comment:
As above, including the Mercury order ID would help with debugging and resolution.